src/Deprecated.jl

```diff
@@ -1,13 +1,57 @@
 ##==============================================================================
 # deprecation staging area
 ##==============================================================================
+##==============================================================================
+## Deprecated in v0.9 Remove in the v0.10 cycle
+##==============================================================================
+
+# temporary promote with warning
+Base.promote_rule(::Type{DateTime}, ::Type{ZonedDateTime}) = DateTime
+function Base.convert(::Type{DateTime}, ts::ZonedDateTime)
+    @warn "DFG now uses ZonedDateTime, temporary promoting and converting to DateTime local time"
+    return DateTime(ts, Local)
+end
+
+@deprecate listSolvekeys(x...) listSolveKeys(x...)
+
+
+
+export InferenceType
+export FunctorSingleton, FunctorPairwise, FunctorPairwiseMinimize
+
+abstract type InferenceType end
+
+# These will become AbstractPrior, AbstractRelativeFactor, and AbstractRelativeFactorMinimize in 0.9.
+abstract type FunctorSingleton <: FunctorInferenceType end
+abstract type FunctorPairwise <: FunctorInferenceType end
+abstract type FunctorPairwiseMinimize <: FunctorInferenceType end
```

Comment:
@dehann If you can check if these are ok to remove now.

Reply:
Could you please help check.  Agree that these should not be in v0.10.

Reply:
Ok, I removed them permanently (again). I couldn't find references in downstream packages.